styleguide/source/_patterns/02-organisms/06-topic/04-topic-related-content/topic-related-content.twig

```diff
@@ -0,0 +1,4 @@
+<div class="topic_related-content">
+  {% include 'atoms-h5' with { 'content': 'Lorem ipsum', 'class': 'topic_related-content_title' } %}
```

Comment:
There should also be a 1px solid gray border/`<hr>` underneath the Topic Label element (you can see it in the wires). Sorry if I missed this in the visual reqs!